src/backends/torch/torchlib.cc

```diff
@@ -725,7 +731,13 @@ namespace dd
                                              + e.what());
               }
 
-            Tensor loss = tloss.loss(y_pred, y, in_vals);
+            Tensor loss;
+            if (_loss == "model")
```

Comment:
Since the template is known, and if this is the only correct loss option for these object detection models, I believe we should set it automatically.